<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerCostTable</class>
 <widget class="QWidget" name="LayerCostTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="txSearch">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 18px &quot;Microsoft JhengHei UI&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>33</height>
            </size>
           </property>
           <property name="text">
            <string>清除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearch">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>33</height>
            </size>
           </property>
           <property name="text">
            <string>查詢</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wBody" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0" columnstretch="1,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0" rowspan="3" colspan="2">
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,2" columnstretch="1,2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0" rowspan="2">
             <widget class="QWidget" name="widget_2" native="true">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>100</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>270</width>
                <height>16777215</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>遊戲列表:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="tbGame">
                 <property name="maximumSize">
                  <size>
                   <width>260</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>遊戲代碼</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>遊戲名稱</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="widget_5" native="true">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_5" columnstretch="1">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0" rowspan="2">
                <widget class="QTabWidget" name="tabWidget">
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <widget class="QWidget" name="page0">
                  <attribute name="title">
                   <string>價目表</string>
                  </attribute>
                  <layout class="QGridLayout" name="gridLayout_8">
                   <item row="1" column="1">
                    <widget class="QPushButton" name="btnCopy">
                     <property name="minimumSize">
                      <size>
                       <width>100</width>
                       <height>33</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>100</width>
                       <height>33</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>複製文字</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <spacer name="verticalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="0" column="0" rowspan="2">
                    <widget class="QPlainTextEdit" name="txt"/>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page1">
                  <attribute name="title">
                   <string>編輯</string>
                  </attribute>
                  <layout class="QGridLayout" name="gridLayout_6">
                   <item row="1" column="1">
                    <widget class="QPushButton" name="btnSaveText">
                     <property name="text">
                      <string>儲存</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="0" column="0" rowspan="2">
                    <widget class="QTextEdit" name="txEdit">
                     <property name="placeholderText">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page2">
                  <attribute name="title">
                   <string>說明</string>
                  </attribute>
                  <layout class="QGridLayout" name="gridLayout_7">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>%表示聯動資料代號，
 例如:
  %Item1  為商品1的名稱  
  %USD1 為商品1的美金售價 
 其它
 %User 為複製文字的人 
 %Time 複製時的時間      </string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>商品: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="tbGameItem">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>110</number>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>商品名稱</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>新台幣</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>美金</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>港幣</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>人民幣</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>林吉特</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>新加坡元</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
